SwerveControllerCommand/constants.py

```diff
@@ -0,0 +1,93 @@
+#
+# Copyright (c) FIRST and other WPILib contributors.
+# Open Source Software; you can modify and/or share it under the terms of
+# the WPILib BSD license file in the root directory of this project.
+#
+
+from wpilib import TimedRobot
+from wpimath.geometry import Translation2d
+from wpimath.kinematics import SwerveDrive4Kinematics
+from wpimath.trajectory import TrapezoidProfile
+import math
+
+
+class Constants:
```

Comment:
We usually don't have a top-level `Constants` class – the module itself is sufficient for namespacing. See https://github.com/robotpy/examples/blob/main/FrisbeeBot/constants.py for another example.